ee/vellum_ee/workflows/tests/test_server.py

```diff
@@ -91,19 +85,27 @@ def test_load_from_module__ts_code_in_file_loader(
     # AND a workflow module with only a code execution node
     files = {
         "__init__.py": "",
-        "workflow.py": """\
+        "workflow.py": """
 from vellum.workflows import BaseWorkflow
 from .nodes.code_execution_node import CodeExecutionNode
 
 class Workflow(BaseWorkflow):
     graph = CodeExecutionNode
+
+    class Outputs(BaseWorkflow.Outputs):
+       final_output = CodeExecutionNode.Outputs.result
 """,
-        "nodes/__init__.py": """\
+        "nodes/__init__.py": """
+from .code_execution_node import CodeExecutionNode
+
+__all__ = ["CodeExecutionNode"]
+""",
+        "nodes/code_execution_node/__init__.py": """
 from .code_execution_node import CodeExecutionNode
 
 __all__ = ["CodeExecutionNode"]
 """,
```

Comment:
[nit] we actually don't code gen this today and put the node in the __init__.py file currently